components/Themed.tsx

```diff
@@ -101,6 +102,16 @@ export function View(props: ViewProps) {
   return <DefaultView style={[{ backgroundColor }, style]} {...otherProps} />;
 }
 
+export function SafeAreaView(props: ViewProps) {
```

Comment:
https://reactnative.dev/docs/safeareaview

Looks like this only applies to IOS. Is this still safe to use with android? Maybe I should just "build" it and try it out

Reply:
Yeah my understanding is this is to help out with devices with "nothces" - i dont think androids have those?